<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>szyfrBox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Szyfrowanie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>441</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabDropbox">
     <attribute name="title">
      <string>Ustawienia konta</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Konto Dropbox</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>160</width>
         <height>102</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Autoryzuj</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>251</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Klucz</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>191</width>
         <height>99</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="labelKlucz">
            <property name="text">
             <string>Twój klucz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditKlucz"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Zapisz</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSpinBox" name="spinBoxOdswiez">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>450</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Odświeżanie</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Klucze</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>19</y>
        <width>381</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Klucz</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Zapisz</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="AESGeneruj">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>AES Generuj</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="KluczAES_line">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>450</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>KLUCZ AES</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Wyślij klucz</string>
     </attribute>
     <widget class="QGroupBox" name="email_box">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>GMAIL</string>
      </property>
      <widget class="QLineEdit" name="email_nadawca">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="email_adresat">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="email_haslo">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_nadawca">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>           Nadawca</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_haslo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>                 Haslo</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_adresat">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>             Adresat</string>
       </property>
      </widget>
      <widget class="QPushButton" name="emailWyslij">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Wyslij</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email_klucz">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_klucz">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>                  Klucz</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="protokoly_box">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Protokoly</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QRadioButton" name="SSL_radio">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>SSL</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="TSL_radio">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>TSL</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="emailRSAGeneruj">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>150</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>RSA Generuj</string>
       </property>
      </widget>
      <widget class="QPushButton" name="aesWyslij">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>180</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Wyslij AES</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="niezaszyfrowanyRSA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PLIK Z KLUCZEM PUBLICZNYM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="szyfrujRSA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Szyfruj RSA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deszyfrujRSA">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Deszyfruj RSA</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nazwaUzytkownikaLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>NAZWA UZYTKOWNIKA</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_szyfrowanie">
     <attribute name="title">
      <string>Szyfrowanie</string>
     </attribute>
     <widget class="QLineEdit" name="nazwaPliku_szyfrowanie">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="szyfrowanieAES_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Szyfruj</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nazwaPliku_deszyfrowanie">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deszyfrowanieAES_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Deszyfruj</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
